scenarios/Assistants/multi-agent_example/README.md

```diff
@@ -0,0 +1,139 @@
+
+The importance of agents in AI cannot be overstated. They are the harbingers of a future where AI systems can interact with, learn from, and adapt to their environments, leading to more personalized and efficient solutions across industries. However, the complexity of creating such intelligent systems has been a significant barrier to their widespread adoption.
+
+Imagine a multi-agent system where each agent is not only equipped with the ability to learn and adapt but also to communicate with other agents persistently and access a vast array of resources without the constraints of context windows. With the Azure Assistant API, developers can create multi-agent systems that are persistent and infinitely long, and append messages as users and agents interact, all while the Assistants accesses files and tools in parallel, including a Code Interpreter and function calling capabilities.
+
+In this blog post as we explore the transformative possibilities of building a multi-agent framework with the Azure Assistant API, a leap forward in making AI more accessible, customizable, and effective for developers and organizations alike. Whether you're an experienced developer or just starting, the Azure Assistant API is a tool that promises to revolutionize the way we think about and implement intelligent agents in our systems.  
+
+In the forthcoming sections of this blog post, we will delve into the architectural blueprint of constructing a robust multi-agent system utilizing the Azure OpenAI Assistant API. Our focus will be on the patterns that serve as the foundation for these intricate frameworks, rather than providing exhaustive code samples. By dissecting these patterns, we aim to equip you with a comprehensive understanding of the principles and strategies essential for orchestrating multiple agents into a cohesive and dynamic system.
+The diagram above serves as an example to demonstrate the overall architectural blueprint. In this example, the Proxy Agent serves as the main orchestrator between the Image Creator Assistant and the Vision Analyzer Assistant. The proxy agent’s function is to send messages across agents to streamline the dialogue between the user and specialized agents within the group chat and maintain ongoing communication with them to guarantee the user's task is carried out to completion.
+
+We define the three agents as follows:
+
+<pre lang=lisp>
+  dalle_assistant = client.beta.assistants.create(
+    name=name_dl,
+    instructions=instructions_dl,
+    model=api_deployment_name,
+    tools=tools
+)
+</pre>
+<pre lang=lisp>
+  vision_assistant = client.beta.assistants.create(
+    name=name_vs,
+    instructions=instructions_vs,
+    model=api_deployment_name,
+    tools=tools
+)
+</pre>
+<pre lang=lisp>
+  user_proxy = client.beta.assistants.create(
+    name=name_pa,
+    instructions=instructions_pa,
+    model=api_deployment_name,
+    tools=tools
+)
+</pre>
```

Comment:
These codeblocks seem wrong?

`<pre></pre>` doesn't seem to render syntax highlighting on github, and the code in the block isn't lisp.